webapp/app.py

```diff
@@ -143,4 +168,26 @@ def login_handler():
 @app.route("/demo")
 @login_required
 def demo():
+    flask.session["invitation_code"] = request.args.get("invitation_code")
+    authentication_token = flask.session["authentication_token"]
+    authorization_header = {
+        "Authorization": f"macaroon root={authentication_token}"
+    }
+    response = requests.get(
```

Comment:
Talked with you directly about it but can you use the `_api_request` function. :smile: